pkg/apiserver/webhooks/datavolume-validate.go

```diff
@@ -271,7 +272,7 @@ func (wh *dataVolumeValidatingWebhook) Admit(ar v1beta1.AdmissionReview) *v1beta
 			return toAdmissionResponseError(err)
 		}
 
-		if !reflect.DeepEqual(dv.Spec, oldDV.Spec) {
+		if !apiequality.Semantic.DeepEqual(dv.Spec, oldDV.Spec) {
```

Comment:
may want to add a couple unit tests to confirm this works as expected

Reply:
Ok, utests failing when they try to do a real change of the spec?

Reply:
Yeah succeeding if resource values are the same but in different units.  Failing if values different.

Reply:
@mhenriks no way to submit DV update request with different units, so submitted with different format (BinarySI and DecimalSI) which fails before the patch and passes with it.